.gitignore

```diff
@@ -1,20 +1,27 @@
-pkg/
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.

Reply:
I can appreciate and understand your concern. I have learned a great deal from this effort. This PR died with the words, "This isn't a pdk module.". I will start again with the lessons I learned. If I am still able to run accepted tests locally from the current master, I will try again to incorporate my changes into a "VP" module. Thanks to all for the feedback.

Reply:
I understand that you are frustrated, but we support over 130 Puppet modules. Modulesync existed long before PDK and is used to rollout changes to all these modules. 

I admit that the ```README.md``` is not up-to-date, but you can use Beaker and Docker to test this module locally:
* [CONTRIBUTING.md](https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.github/CONTRIBUTING.md.erb)

Reply:
Frustration is my norm. :) Hopefully, my most recent attempt will be successful.

Reply:
No problem. As you can see, I have created several issue to support you regarding RHEL and Ubuntu releases etc. 

I will try to do this in several pull requests to keep it simple. ;)